src/main/java/webserver/handler/MappingHandler.java

```diff
@@ -0,0 +1,24 @@
+package webserver.handler;
+
+import webserver.servlet.FileServlet;
+import webserver.servlet.HomeServlet;
+import webserver.servlet.HttpServlet;
+import webserver.servlet.UserCreateServlet;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class MappingHandler {
+    private static Map<String, Object> servlets = new HashMap<>();
+
+    static {
+        servlets.put("/", new HomeServlet());
+        servlets.put("/user/create", new UserCreateServlet());
+    }
+
+    public static HttpServlet getServlets(String absPath) {
+        Optional path = Optional.ofNullable(servlets.get(absPath));
+        return (HttpServlet) path.orElse(new FileServlet());
```

Comment:
```suggestion
        return (HttpServlet) servlets.getOrDefault(absPath, new FileServlet());
```
getOrDefault를 써보면 어떨까요?

Reply:
반영했습니다!